### DISCUSSION TITLE
Certificato competenze non visualizzato dai genitori/alunni

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#525] [Richieste di aiuto] [answered]
Buonasera , come mai i genitori degli alunni del biennio non riescono a visualizzare nel registro i certificati di competenze elaborate in sede di scrutinio?

**Comment by trinko** [comment, marked as answer]
Il certificato delle competenze è gestito attualmente dalla segreteria, per cui l'alunno non lo può scaricare direttamente.
Ma poiché questa è una modifica semplice e dovevo già correggere qualcosa nella pagina di visualizzazione degli esiti vista dagli alunni/genitori, penso che la farò tra qualche giorno.

**Comment by trinko** [comment]
Se ora effettui l'aggiornamento da **SISTEMA > AGGIORNAMENTO** e carichi le ultime modifiche avrai la gestione richiesta.

Per renderla operativa, vai su **SISTEMA > PARAMETRI** e imposta a **S** il nuovo parametro **scrutinio_genitori_competenze** che trovi nella sezione SISTEMA.

**Reply by sempliciniluigi89** [reply]
Grazie mille,Adesso il  dirigente scolastico ha rilevato un problema che se non erro ,ho evidenziato l'anno scorso ,ossia :sul certificato delle competenze manca il numero (N°.........)il che è un problema ,in quanto non rende il certificato agli atti ,quindi non sarà mai ufficiale in quanto privo di un numero.

**Reply by sempliciniluigi89** [reply]
IGrazie mille,Adesso il  dirigente scolastico ha rilevato un problema che se non erro ,ho evidenziato l'anno scorso ,ossia :sul certificato delle competenze manca il numero (N°.........)il che è un problema ,in quanto non rende il certificato agli atti ,quindi non sarà mai ufficiale in quanto privo di un numero.

**Reply by trinko** [reply]
Capisco. Al momento, però, non posso fare questa modifica.
